.env

```diff
@@ -1,3 +1,5 @@
 VUE_APP_BACKEND_HOST=https://tradeshare-backend-for-dev.herokuapp.com/api/v1
 #VUE_APP_BACKEND_HOST=https://tradeshare-backend-for-test.herokuapp.com/api/v1
-VUE_APP_ALPACA_INTEGRATION_URL=https://app.alpaca.markets/oauth/authorize?response_type=code&client_id=1b32c84697cc9bc9c95f646e487c02e7&redirect_uri=http://127.0.0.1:8081/confirm
\ No newline at end of file
+
+VUE_APP_ALPACA_INTEGRATION_URL=https://app.alpaca.markets/oauth/authorize?response_type=code&client_id=1b32c84697cc9bc9c95f646e487c02e7&redirect_uri=http://127.0.0.1:8081/confirm
```

Comment:
Just a reminder to add this URL into Heroku's secret (use this URL instead: `https://app.alpaca.markets/oauth/authorize?response_type=code&client_id=1b32c84697cc9bc9c95f646e487c02e7&redirect_uri=https://tradeshare-dev.herokuapp.com/confirm`)